tests/trainers/test_regression.py

```diff
@@ -30,6 +32,8 @@ def test_trainer(self, name: str, classname: Type[LightningDataModule]) -> None:
         model_kwargs = conf_dict["module"]
         model = RegressionTask(**model_kwargs)
 
+        model.model = RegressionTestModel()
```

Comment:
I could monkeypatch `torchvision.models.resnet18` here to save having the ResNet18 object actually being created in the constructor, but it seems like that won't save much time/RAM and will change when we upgrade RegressionTask